### PR TITLE
Update readme to reflect changes to project structure and api

### PR DESCRIPTION
**Pull request by CalebABG**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,37 +73,39 @@ Silk uses [PostgreSQL](https://www.postgresql.org/) for its backend database sto
    - ```
      <UserSecretsId>VelvetThePanda-SilkBot</UserSecretsId>
      ```
-2. The location of the ```secrets``` file is different between Operating Systems, but in a system-protected user profile folder on your computer.
+2. The location of the ```secrets.json``` file is different between Operating Systems, but in a system-protected user profile folder on your computer.
```
Comment (VelvetToroyashi) [inline README.md]: Shouldn't this be inline codeblock (\`\`) instead of a full codeblock (\`\`\`\`\`\`)?

File: README.md [hunk 2]
```diff
@@ -94,4 +116,4 @@ To run Silk! as a docker container, you can simply create an `appSettings.json`
 Both Postgres and Silk! will have to initialize on the first run, which may cause slightly degraded startup times as Silk! creates requisite tables on the database. 
 
 Further startup times should be no more than ~3 seconds to fully initialize and do cache runs.
-![](https://files.velvetthepanda.dev/silk.png)
+![](https://files.velvetthepanda.dev/silk.png)
```
Comment (VelvetToroyashi) [inline README.md:119]: Git...Why

**Review by VelvetToroyashi — CHANGES_REQUESTED** [review]
LGTM, See review comments

File: README.md (hunk 1 above)
Comment (CalebABG) [reply README.md]: Yup will change those that are like that.

**Pull request merged**